website/docs/configuration/functions/jsonencode.html.md

```diff
@@ -37,6 +37,11 @@ types, passing the `jsonencode` result to `jsondecode` will not produce an
 identical value, but the automatic type conversion rules mean that this is
 rarely a problem in practice.
 
+-> **Note:** When encoding strings, this function escapes some characters using
```

Comment:
I like the text here but I feel like this is too minor a point to justify a callout box, because callout boxes are designed to make something appear more important than what's around it, but to me this feels considerably less important than the other content on this page. (It's a mostly-inconsequential implementation quirk that most users will never encounter.)

I suggest dropping the `-> **Note:**` from the front of this and then keeping everything else you wrote here -- which is great -- as a normal paragraph.

Reply:
Thanks for that suggestion, I completely agree.